python/mops-torch/tests/sasaw.py

```diff
@@ -51,35 +57,59 @@ def test_sasaw():
     assert torch.allclose(reference, actual)
 
 
-def test_sasaw_grad():
-    A = torch.rand(70, 10, dtype=torch.float64, requires_grad=True)
-    B = torch.rand(70, 50, dtype=torch.float64, requires_grad=True)
-    W = torch.rand(3, 5, 50, dtype=torch.float64, requires_grad=True)
-    C = torch.rand(50, dtype=torch.float64)
-    indices_output_1 = torch.randint(3, size=(70,), dtype=torch.int32)
-    indices_W_1 = torch.randint(3, size=(70,), dtype=torch.int32)
-    output_size_2 = 15
-    indices_A = torch.randint(10, size=(50,), dtype=torch.int32)
-    indices_W_2 = torch.randint(5, size=(50,), dtype=torch.int32)
-    indices_output_2 = torch.randint(output_size_2, size=(50,), dtype=torch.int32)
+@pytest.mark.parametrize("dtype", [torch.float64])
```

Comment:
We don't need to parametrize if there is a single value, just use this dtype in the test code